src/core/config/styles.js

```diff
@@ -56,6 +58,14 @@ export function getCurrentTheme(isContrastMode) {
     const palette = getCurrentPalette(isContrastMode);
     return createMuiTheme({
         palette: palette,
+        typography: {
+            fontFamily: [
+                '"HK Grotesk"',
+                'Arial',
+                'sans-serif',
+            ].join(','),
```

Comment:
A bit overkill IMO. Why not join it yourself?

Reply:
Done